HealthcareSystem/DoctorFrontend/AddFeedback.xaml.cs

```diff
@@ -28,35 +28,6 @@ public AddFeedback()
 
         private void Button_Click(object sender, RoutedEventArgs e)
         {
```

Comment:
Since the Commentator property in Feedback can only be a Patient, I don't really see how it would be possible for other users to leave feedback at all at this point. In light of this I would remove the option to leave feedback from the doctor, secretary, and manager user interfaces for now, rather than leave the interface unchanged and make the send button do nothing.